src/cascadia/TerminalSettingsModel/ActionArgs.idl

```diff
@@ -203,6 +203,8 @@ namespace Microsoft.Terminal.Settings.Model
 
     [default_interface] runtimeclass SendInputArgs : IActionArgs
     {
+        SendInputArgs(String input);

```

Comment:
Mentioned in the spec, but I am *very much* not sold on the currency being _sendInput actions_

Reply:
Fair